app/models/answer.rb

```diff
@@ -1,4 +1,8 @@
 class Answer < ApplicationRecord
   belongs_to :user
   belongs_to :question
+
+  validates :user, presence: true
+  validates :question, presence: true
+  validates :content, length: { maximum: 100 }
```

Comment:
As you mentioned I think the maximum lengths on the content fields can be increased to possible ~300 or 400. There may be some subject questions that are subjective in nature that warrant a longer response?

Reply:
sounds good, let me update